store/tikv/region_cache_test.go

```diff
@@ -299,3 +299,21 @@ func (s *testRegionCacheSuite) TestUpdateStoreAddr(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(getVal, BytesEquals, testValue)
 }
+
+func (s *testRegionCacheSuite) TestListRegionIDsInCache(c *C) {
+	// ['' - 'm' - 'z']
+	region2 := s.cluster.AllocID()
+	newPeers := s.cluster.AllocIDs(2)
+	s.cluster.Split(s.region1, region2, []byte("m"), newPeers, newPeers[0])
+
+	regionIDs, err := s.cache.ListRegionIDsInKeyRange(s.bo, []byte("a"), []byte("z"))
+	c.Assert(err, IsNil)
+	c.Assert(len(regionIDs), Equals, 2)
```

Comment:
Why only check length? I suppose you can check the IDs are correct.
`c.Assert(regionIDs, DeeqEquals, []uint64{s.region1, region2})`